admin_tests/pre_reg/test_views.py

```diff
@@ -1,110 +0,0 @@
-# -*- coding: utf-8 -*-
-import httplib as http
-from modularodm import Q
-
-from django.core.urlresolvers import reverse
-from django.contrib.auth.models import Group
-
-from nose.tools import *  # flake8: noqa
-
-from tests import factories
-from tests.base import fake, AdminTestCase
-
-from website.models import MetaSchema
-from website.project.model import ensure_schemas
-
-from admin.common_auth.models import MyUser as User
-from admin.pre_reg.views import *  # noqa
-
-PREREG_GROUP = Group.objects.get(name='prereg_group')
-
-
-class PreregViewsTests(AdminTestCase):
```

Comment:
It didn't actually run and wasn't included in Travis for that reason. I commented it out, but it was pointed out that we could just find it in the history of commits and just to delete it for now.